### ISSUE TITLE
Implement take with more commands

### ISSUE THREAD

**Issue by AlphaBeta906** [R1DF/Protests-Simulator-2022#1]
One feature of the game I really despised was the take command, which was really a hassle. Problems include:

- No take all command 
- No multiple-take ("apple 1 potato 1 dice of fate 1")
- Take command ends prematurely (same with other commands like use)

**Comment by R1DF** [comment]
Fair enough. I'll see what I can do when I update this project. All of the stuff you suggested is definitely possible, but an update isn't planned for now. I do want to eventually make one, though.
Wouldn't call it a "feature" if you despised it, though. ¯\_(ツ)_/¯
